Nim In Action test failures on OpenBSD

Some of the `Nim In Action` tests fail on OpenBSD.

- The `SDL` tests only specify `libName` for Linux, Mac and Windows: https://github.com/nim-lang/Nim/blob/devel/tests/niminaction/Chapter8/sdl/sdl.nim#L1
- The `SFML` tests fail with the following error:

```
Category: niminaction
Name: tests/niminaction/Chapter8/sfml/sfml_test.nim C++ --nilseqs:on
Action: compile
Result: reNimcCrash
-------- Expected -------

--------- Given  --------
$ /home/administrator/build/Nim/bin/nim cpp --hints:on -d:testing --nimblePath:tests/deps --nilseqs:on --nimCache:nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f831a39fe22ef81cd6a2f25664 --stdout '--hint[Path]:off' '--hint[Processing]:off' tests/niminaction/Chapter8/sfml/sfml_test.nim
Hint: used config file '/home/administrator/build/Nim/config/nim.cfg' [Conf]
Hint: used config file '/home/administrator/build/Nim/config/config.nims' [Conf]
Hint: used config file '/home/administrator/build/Nim/tests/config.nims' [Conf]
CC: stdlib_system.nim
CC: stdlib_posix.nim
CC: stdlib_times.nim
CC: stdlib_os.nim
CC: sfml_test.nim
Error: execution of an external compiler program 'clang++ -c  -w  -I/home/administrator/build/Nim/lib -I/home/administrator/build/Nim/tests/niminaction/Chapter8/sfml -o /home/administrator/build/Nim/nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f831a39fe22ef81cd6a2f25664/sfml__test.nim.cpp.o /home/administrator/build/Nim/nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f831a39fe22ef81cd6a2f25664/sfml__test.nim.cpp' failed with exit code: 1

/home/administrator/build/Nim/nimcache/tests/niminaction/Chapter8/sfml/sfml_test.nim_d8de44f831a39fe22ef81cd6a2f25664/sfml__test.nim.cpp:10:10: fatal error: 'SFML/Graphics.hpp' file not found
#include <SFML/Graphics.hpp>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
```

--- a/tests/niminaction/Chapter8/sdl/sdl.nim
+++ b/tests/niminaction/Chapter8/sdl/sdl.nim
@@ -4,6 +4,12 @@ elif defined(Linux):
   const libName* = "libSDL2.so"
 elif defined(MacOsX):
   const libName* = "libSDL2.dylib"
+elif defined(openbsd):
+  const libName* = "libSDL.so.8.0"
+elif defined(freebsd):
+  const libName* = "libSDL.so"
+else:
+  {.error: "SDL library name not set for this platform".}
 
 type
   SdlWindow = object

--- a/tests/niminaction/Chapter8/sdl/sdl.nim
+++ b/tests/niminaction/Chapter8/sdl/sdl.nim
@@ -1,13 +1,11 @@
 when defined(Windows):
   const libName* = "SDL2.dll"
-elif defined(Linux):
+elif defined(Linux) or defined(freebsd):
   const libName* = "libSDL2.so"
 elif defined(MacOsX):
   const libName* = "libSDL2.dylib"
 elif defined(openbsd):
-  const libName* = "libSDL.so.8.0"
-elif defined(freebsd):
-  const libName* = "libSDL.so"
+  const libName* = "libSDL2.so.0.6"
 else:
   {.error: "SDL library name not set for this platform".}
 

